sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -238,7 +238,7 @@ async def create_queue(self, name: str, **kwargs) -> QueueProperties:
             dead_lettering_on_message_expiration=kwargs.pop("dead_lettering_on_message_expiration", None),
             default_message_time_to_live=kwargs.pop("default_message_time_to_live", None),
             duplicate_detection_history_time_window=kwargs.pop("duplicate_detection_history_time_window", None),
-            entity_availability_status=kwargs.pop("entity_availability_status", None),
+            availability_status=None,
```

Comment:
for confirmation: it's not a settable property on the entity?

Reply:
Correct++

Reply:
@KieranBrantnerMagee so it means the kwargs pop was a mistake?

Reply:
"Mistake" isn't the word I'd use.  It should not be settable; it's effectively a read-only setting although it's theoretically exposed on the parameters that can be passed to create.  (If this is what you're getting at, it does beg the question of whether it should be addressed in swagger; I made some assumptions that I might have to reassess if you would think it would)